editor/src/core/shared/element-template.ts

```diff
@@ -1929,6 +1929,29 @@ export function getElementsByUIDFromTopLevelElements(
 
 export type ChildOrAttribute = JSXElementChild | JSXAttribute
 
+export type ChildOrAttributeWithBranch = ChildOrAttribute & { branch: boolean }
```

Comment:
I think we can avoid needing this boolean by using some code from `conditionals.ts` (with a tiny bit of tweaking) to get the last part of the path for both clauses of the conditional. We should be able to take the logic from `getConditionalClausePath` for that.

Reply:
this should do it! https://github.com/concrete-utopia/utopia/pull/3352/commits/2322fb32339dc44f84032f481c17ad9272ce18ed